scripts/install-fabric.sh

```diff
@@ -50,9 +48,9 @@ begins_with_short_option()
 print_help()
 {
 	printf 'Usage: %s [-f|--fabric-version <arg>] [-c|--ca-version <arg>] <comp-1> [<comp-2>] ... [<comp-n>] ...\n' "$0"
-	printf '\t%s\n' "<comp>: Component to install one or more of  d[ocker]|b[inary]|s[amples]. If none specified, all will be installed"
-	printf '\t%s\n' "-f, --fabric-version: FabricVersion (default: '2.4.3')"
-	printf '\t%s\n' "-c, --ca-version: Fabric CA Version (default: '1.5.3')"
+	printf '\t%s\n' "<comp> Component to install, one or more of  docker | binary | samples | podman  First letter of component also accepted; If none specified docker | binary | samples is assumed"
```

Comment:
Seeing this too late... but "podman" isn't a component to install, it is HOW to pull the images from docker hub, right?

Reply:
yes you are correct... see cmt below...